### ISSUE TITLE
Incompatible with bootsnap.

### ISSUE THREAD

**Issue by konalegi** [danmayer/coverband#498]
**Describe the bug**
It's not a really bug, but according the code below Coverband is not compatible with Bootsnap. https://github.com/Shopify/bootsnap/blob/main/lib/bootsnap/compile_cache/iseq.rb#L87. 

I believe a lot of people are using Bootsnap these days and specifically in big projects. It will be nice to add this info to the "Known Issues" section. 

Thanks!

**Comment by danmayer** [comment]
Thanks yeah this might be worth noting. I think bootsnap is mostly for development and coverband primarily is intended for production observability... I guess some use it to speed up deploys as well. I can't think of a real work around for this other than updating the docs as you mentioned. 

Let me know if you have any other ideas otherwise I will update the docs

**Comment by konalegi** [comment]
@danmayer I've spent a lot of time trying to make it work together, but the only possibility that I've found is to start Coverband with enabled Coverage on the part of the hosts only. It might be the solution for others. 

> I think bootsnap is mostly for development and coverband primarily is intended for production observability... I guess some use it to speed up deploys as well

I don't agree here, since in modern env when a lot of stuff is running in containers and they dynamically brought up on increased traffic, start-up time becomes very important.
We are using our application in k8s, and starting time is very important for the HPA (horizontal pod autoscaling), when we need to bring pods when the traffic starts increasing as fast as possible. For instance, our app with Coverband enabled starts in ~50s without it ~37s, which is pretty significant.

> Let me know if you have any other ideas otherwise I will update the docs

Yeah, no other ideas, I agree that only we can do is to update docs and everyone will decide what is more important.
